exporter/coralogixexporter/README.md

```diff
@@ -0,0 +1,45 @@
+# Coralogix Exporter
+
+The Coralogix exporter sends traces to [Coralogix](https://coralogix.com/) as
+Coralogix logs.
+
+Supported pipeline types: traces 
+
+> Please review the Collector's [security
+> documentation](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/security.md),
+> which contains recommendations on securing sensitive information such as the
+> API key required by this exporter.
+
+## Configuration
+
+Example configuration:
+```yaml
+exporters:
+  coralogix:
+    # The Coralogix logs ingress endpoint
+    # May also be specified with the CORALOGIX_ENDPOINT environment variable
+    endpoint: "https://api.coralogix.com"
+
+    # Your Coralogix private key (sensitive)
+    # May also be specified with the CORALOGIX_PRIVATE_KEY environment variable
```

Comment:
I assume this functionality (and the rest of environment variables) will be added on a future PR?

Reply:
Yes, this functionality will be added on a future PR.